BioArchLinux/agat/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Bipin Kumar <bipin@ccmb.res.in>
+
+pkgname=agat
+_pkgname=AGAT
+pkgver=1.0.0
+pkgrel=1
+pkgdesc="Another Gtf/Gff Analysis Toolkit: Suite of tools to handle gene annotations in any GTF/GFF format. https://www.doi.org/10.5281/zenodo.3552717"
```

Comment:
It would be better to make the `pkgdesc` shorter, maybe remove the doi URL?

Reply:
according to bioarchlinux  description [here](https://f1000research.com/posters/11-809) DOIs are used to query and cite packages used,

>The repository provides, where possible, a DOI in each package description. This makes it easy for users to find out more about each package’s use and methodology

i was just following that guideline. if you wish i will remove the DOI